pkg/scheduler/framework/interface.go

```diff
@@ -205,21 +205,21 @@ func (s *Status) Message() string {
 	return strings.Join(s.Reasons(), ", ")
 }
 
-// SetFailedPlugin sets the given plugin name to s.failedPlugin.
-func (s *Status) SetFailedPlugin(plugin string) {
-	s.failedPlugin = plugin
+// SetAbortPlugin sets the given plugin name to s.abortPlugin.
```

Comment:
Would you mind marking the old method as deprecated and then adding the new method? 
IMO, we need to consider the migration process of our users.

Reply:
@alculquicondor Do we usually keep old functions like this? 
If we do such a deprecation process for all breaking changes in the scheduler, it'd be burden for maintenance more or less.